spring-boot-weave-service-discovery/README.md

```diff
@@ -1,10 +1,8 @@
 ---
 layout: guides
 title: "Service Discovery with Weave for Dockerized Spring Apps"
-description: |
-  In this example we will demonstrate how Weave allows you to quickly deploy a containerised
-  microservice based Java application, developed in Spring with simple service discovery using
-  Weave Run. No modifications are required to your application.
+description: Use Weave to quickly deploy a Java-based containerized
+microservice developed in Spring with simple service discovery using Weave Run.
```

Comment:
@abuehrle I think these need to be done either in single line or using the `|` syntax, i.e.
```yaml
description: |
  Use Weave to quickly deploy a Java-based containerized
  In this example we will demonstrate how Weave allows you to quickly deploy a containerised
  microservice developed in Spring with simple service discovery using Weave Run.
```